spec/services/stats/r003_business_unit_performance_report_spec.rb

```diff
@@ -212,22 +212,23 @@ module Stats
       describe '#to_csv' do
         let(:report_csv) do
           Timecop.freeze Time.new(2017, 6, 30, 12, 0, 0) do
-            report = R003BusinessUnitPerformanceReport.new; report.run; report.to_csv
+            d1 = Date.new(2017, 6, 4)
+            d2 = Date.new(2017, 6, 30)  
+            report = R003BusinessUnitPerformanceReport.new(period_start: d1, period_end: d2); report.run; report.to_csv
```

Comment:
I think its better to have statements on multiple lines if we can - easier to read and using the `;` syntax to brake up statements is not really very common in Ruby code 👍

Reply:
thank for this, I didn't really pay attention to this line other than changing the time frame,  good to know `;` usage in Ruby :)